interactions/client/client.py

```diff
@@ -867,7 +877,9 @@ async def _on_websocket_ready(self, event: events.RawGatewayEvent) -> None:
         self._user._add_guilds(expected_guilds)
 
         if not self._startup:
-            while len(self.guilds) != len(expected_guilds):
+            while len(self.guilds) != len(
+                expected_guilds
+            ):  # TODO: potentially use self.guild_count instead of len(self.guilds)
```

Comment:
We explictly don't want to use guild_count here, as this behaviour relies on unloaded guilds not being listed.

Reply:
Comment has been removed.